py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,40 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
+    format_device = '{name}'
+
+    class Meta:
+        def deprecate_function(config):
+            out = {}
+            if 'format_prefix' in config:
+                out['format'] = u'{}{{format_device}}'.format(config['format_prefix'])
+            return out
```

Comment:
Sure. I can give your patch a go only if you want to whip one up. Have fun tonight. Is there a way to `time` a module (for optimization purpose)?